src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -26,6 +25,11 @@ import { QuickBarParams } from "./show-dialog-quick-bar";
 import { HassEntity } from "home-assistant-js-websocket";
 import { compare } from "../../common/string/compare";
 import { SingleSelectedEvent } from "@material/mwc-list/mwc-list-foundation";
+import memoizeOne from "memoize-one";
+import "../../common/search/search-input";
+import { mdiConsoleLine } from "@mdi/js";
+import { scroll } from "lit-virtualizer";
+import { styleMap } from "lit-html/directives/style-map";
```

Comment:
I see you stopped using the VScode auto organize imports 😜

Reply:
Yep 😄